src/components/Breadcrumb/Breadcrumb.vue

```diff
@@ -0,0 +1,94 @@
+<!--
+ - @copyright Copyright (c) 2020 Raimund Schlüßler <raimund.schluessler@mailbox.org>
+ -
+ - @author Raimund Schlüßler <raimund.schluessler@mailbox.org>
+ -
+ - @license GNU AGPL version 3 or any later version
+ -
+ - This program is free software: you can redistribute it and/or modify
+ - it under the terms of the GNU Affero General Public License as
+ - published by the Free Software Foundation, either version 3 of the
+ - License, or (at your option) any later version.
+ -
+ - This program is distributed in the hope that it will be useful,
+ - but WITHOUT ANY WARRANTY; without even the implied warranty of
+ - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ - GNU Affero General Public License for more details.
+ -
+ - You should have received a copy of the GNU Affero General Public License
+ - along with this program. If not, see <http://www.gnu.org/licenses/>.
+ -
+ -->
+
+<docs>
+
+### General description
+
+This component is meant to be used inside a Breadcrumbs component.
+
+</docs>
+
+<template>
+	<div class="crumb svg folder"
+		draggable="false">
+		<a :href="'#' + path">
+			<span v-if="icon" :class="icon" class="icon" />
+			<span v-else>{{ title }}</span>
+		</a>
+	</div>
+</template>
+
+<script>
+export default {
+	name: 'Breadcrumb',
+	props: {
+		title: {
+			type: String,
+			required: true,
+		},
+		path: {
+			type: String,
+			required: false,
+			default: '',
+		},
+		icon: {
+			type: String,
+			required: false,
+			default: '',
+		},
+	},
+}
+</script>
+
+<style lang="scss" scoped>
+
+.breadcrumb {
+	width: calc(100% - 88px);
```

Comment:
88 px?

Reply:
Should be removed. This is a leftover from how I use the Breadcrumbs in my app.